docs/guides/nrfconnect_android_commissioning.md

```diff
@@ -141,13 +141,12 @@ To form a Thread network, complete the following steps:
 
         $ docker network create --ipv6 --subnet 2001:db8:1::/64 -o com.docker.network.bridge.name=otbr0 otbr
 
-2.  Start the OTBR container using the command below with _rcp-dk_ in the last
-    line replaced with the device node name of the nRF52840 DK that is running
-    the RCP firmware (for example, _/dev/ttyACM0_):
+2.  Start the OTBR container using the device node name of the nRF52840 DK
```

Comment:
I remember last time someone requested the information that the word which should be replaced is in the last line as it wasn't obvious where to look for it.

Reply:
@LuDuda what is your opinion? We were talking about removing `_rcp-dk_`, as Łukasz reported that people had a problem to find it anyway, so the blog post was already modified and I only wanted to align this instruction to blogpost.

Reply:
I'm fine with removing _rcp-dk_ :) I just think we may leave information that `/dev/ttyACM0` in the last line should be replaced.

Reply:
Oh ok, I didn't get it. I will add it.